src/server/report/app.js

```diff
@@ -387,20 +364,29 @@ class App extends LitElement {
 		}
 
 		const browsers = data.browsers.filter(b => this._filterBrowsers.includes(b.name));
+		const browserResults = new Map();
+		tests.forEach(t => {
+			t.results.forEach(r => {
+				if (!browserResults.has(r.name)) {
+					browserResults.set(r.name, 0);
+				}
+				if (r.passed) browserResults.set(r.name, browserResults.get(r.name) + 1);
+			});
+		});
+
 		const selectedBrowser = browsers[this._selectedBrowserIndex] ||
-		browsers.find(b => testData.results.find(r => r.name === b.name && !r.passed)) ||
-		browsers[0];
-		const selectedResult = testData.results.find(r => r.name === selectedBrowser.name);
+			browsers.find(b => browserResults.get(b.name) < tests.length) ||
+			browsers[0];
 
 		const tabs = browsers.map((b) => {
-			const result = testData.results.find(r => r.name === b.name);
+			const numPassed = browserResults.get(b.name);
 			return {
-				content: html`<div style="padding: 20px;">${renderResult(result, { fullMode: this._fullMode, layout: this._layout, showOverlay: this._overlay })}</div>`,
+				content: renderBrowserResults(b, tests, { fullMode: this._fullMode, layout: this._layout, showOverlay: this._overlay }),
 				label: b.name,
 				id: b.name.toLowerCase(),
 				selected: b.name === selectedBrowser.name,
-				status: result.passed ? 'passed' : 'failed',
-				statusType: result.passed ? TAB_STATUS_TYPE.NORMAL : TAB_STATUS_TYPE.ERROR
+				status: `${numPassed}/${tests.length}`,
```

Comment:
```suggestion
				status: `${numPassed}/${tests.length} passed`,
```
I wonder if this would be helpful, or if it's just something you need to get used to.  But when this is red, I couldn't immediately tell if it was telling me number of failed tests or number of passed tests

Reply:
I was wondering about that too -- let's add it for extra clarity and we can always adjust later.

<img width="245" alt="Screenshot 2023-07-19 at 10 23 50 AM" src="https://github.com/BrightspaceUI/testing/assets/5491151/3031e4bf-c341-4f7d-937b-9127ce1e1b13">